api/src/task/ecs_background_task.py

```diff
@@ -47,8 +49,12 @@ def entrypoint(db_session: db.Session):
     def decorator(f: Callable[P, T]) -> Callable[P, T]:
         @wraps(f)
         def wrapper(*args: P.args, **kwargs: P.kwargs) -> T:
+            # Initialize New Relic at the start of any task
+            init_newrelic()
```

Comment:
We'd still keep init where it was, create_app gets called when we start up the API as a service and for background tasks already, and all that happens well before this for background tasks.

Reply:
Makes sense, thanks

Reply:
We shouldn't need this here - create_app already is called so init_newrelic was already called well before this

Reply:
Got it -- forgot that create_app is called on every task well